### DISCUSSION TITLE
SSDP-issue

### DISCUSSION THREAD

**Discussion by KeesKemper** [alexkn/node-red-contrib-dlna#9] [Q&A]
In v0.1.0 I am able to access the renderer with the friendly name. V0.2.1 gives the following error for every DLNA action:
"Error: device url not found".
Someone an idea what can be wrong?

**Comment by alexkn** [comment]
No problems on my system.
Do you see any errors in the logs?

**Reply by KeesKemper** [reply]
It is working now. Maybe some issue with the cache?
